packages/runner-ct/src/app/app.tsx

```diff
@@ -78,6 +93,7 @@ App.propTypes = {
     viewportHeight: PropTypes.number.isRequired,
     viewportWidth: PropTypes.number.isRequired,
   }).isRequired,
+  // @ts-expect-error
```

Comment:
since this is a tsx file now can we just delete these?

Reply:
Not, this directive is extremely useless while migration to typescript, cause it allows to ignore the error and while there is **no error** it will not fail

Reply:
I mean can we just delete the entire `PropTypes` block entirely? It is not very useful since TS typing should be enough.

Reply:
I don't think this is a good idea, cause our typings right now is completely ignored and typescript is not running anywhere

Reply:
Right, you mean we are not doing a `yarn tsc` on CI to confirm the types are valid? We should prfioritize this to get full value from TS. Good idea to leave these until then, thanks